pkg/trace/traceutil/normalize.go

```diff
@@ -185,7 +185,7 @@ func normalize(v string, allowDigitStartChar bool) string {
 			chars++
 		// If it's not a unicode letter, and it's the first char, and digits are allowed for the start char,
 		// we should goto end because the remaining cases are not valid for a start char.
-		case allowDigitStartChar && chars == 0:
+		case removeDigitStartChar && chars == 0:
```

Comment:
If I understand this correctly, the existing behavior worked as expected, despite the flag name having the opposite meaning of what it actually intended, correct?

Reply:
Yeah it seems the behavior was correct, but Cecile Terpin pointed out the name here is confusing